prompts.js

```diff
@@ -1,20 +1,49 @@
+const axios = require('axios');
+
+const http = axios.create({
+  headers: {
+    Authorization: `Bearer ${process.argv[4]}`,
+  },
+  baseURL: 'https://api.staging-chameleon.nsoft.cloud',
+});
+
 const validateBundleName = (name) => {
   if (name.length < 2) {
     return 'Bundle name must contain two or more characters';
+  } else {
```

Comment:
Since we have a `return` statement inside `if`, no need for `else` block here.
Same goes for namespace validation.